tests/topology_test.cpp

```diff
@@ -120,7 +114,43 @@ TEST_CASE("get generation id") {
     REQUIRE(tc->get_generation_id() == 0);
     auto config = load_config("source/http");
     tc->add_config(config);
-    wait(1000);
     REQUIRE(tc->get_generation_id() == 1);
   });
+}
+
+TEST_CASE("test new topology") {
+  auto config = load_config("batch_file_to_file");
+  vectorcxx::test::setup_data();
```

Comment:
I think we could use `TEST_CASE_METHOD(VectoreService, "xxx")` for this purpose, and there is no need to manually setting up.